pkg/authorization/bootstrap/policy.go

```diff
@@ -46,6 +46,21 @@ const (
 	SystemKcpWorkspaceAccessGroup = "system:kcp:workspace:access"
 )
 
+const (
+	// SystemMastersGroup is the group inherited from k8s codebase - all powerful, all knowing!
+	// Users should not be added to this group.
+	SystemMastersGroup = "system:masters"
+)
+
+const (
+	// SystemKcpAdminUser is the user that has all permissions across all workspaces.
+	SystemKcpAdminUser = "kcp-admin"
+	// SystemShardAdmin is the user that has all permissions across all workspaces.
+	SystemShardAdmin = "shard-admin"
```

Comment:
does the name alone has impact?

Reply:
same for kcp-admin.

Reply:
with groups flow. it does not anymore. Will see if worth keeping so we can have less strings in code or just drop this.

Reply:
deleted those.